### DISCUSSION TITLE
New version 0.110.1 removes required package starlette

### DISCUSSION THREAD

**Discussion by nirbar1985** [fastapi/fastapi#11396] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
"""FastAPI framework, high performance, easy to learn, fast to code, ready for production"""

__version__ = "0.110.1"

from starlette import status as status

from .applications import FastAPI as FastAPI
from .background import BackgroundTasks as BackgroundTasks
```

### Description

import of FastAPI fails because starlette package not exists

### Operating System

macOS

### FastAPI Version

0.110.1

### Pydantic Version

1.10.14

### Python Version

3.8.8

**Comment by Kludex** [comment]
Where do you see that?

**Reply by nirbar1985** [reply]
poetry add fastapi==0.110.1
 
 Then i see this - 
 Updating dependencies
Resolving dependencies... (48.5s)

Package operations: 0 installs, 1 update, 1 removal

  • Removing starlette (0.36.3)
  • Updating fastapi (0.110.0 -> 0.110.1)

Writing lock file

**Reply by Kludex** [reply, marked as answer]
I'm not sure what is happening there, but here it works just fine...

**Reply by nirbar1985** [reply]
you are right, there are probably some issues with the consumption from the artifactory
thanks!
